prismic-model/json/collection-venues.json

```diff
@@ -6,18 +6,57 @@
         "label": "Title"
       }
     },
-    "description": {
+    "heading": {
       "type": "StructuredText",
```

Comment:
Have we forgotten to run the JSON serializer?

Reply:
yes .. let me update this !